### PR TITLE
chore(tiering): Lots of metrics

### PR DESCRIPTION
**Pull request by dranikpg**
Add more metrics to tieringv2

### CODE REVIEW

**Comment by dranikpg** [comment]
@romange, are you happy now? 😆

File: src/server/server_family.cc [hunk 1]
```diff
@@ -2056,24 +2047,20 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
...
-    append("tiered_read_latency_usec", m.disk_stats.read_delay_usec);
-    append("tiered_writes", m.tiered_stats.tiered_writes);
-    append("tiered_reserved", m.tiered_stats.storage_reserved);
-    append("tiered_capacity", m.tiered_stats.storage_capacity);
-    append("tiered_aborted_writes", m.tiered_stats.aborted_write_cnt);
-    append("tiered_flush_skipped", m.tiered_stats.flush_skip_cnt);
-    append("tiered_throttled_writes", m.tiered_stats.throttled_write_cnt);
-  }
-
   if (should_enter("TIERED_V2", true)) {
```
Comment (romange) [inline src/server/server_family.cc:2050]: please keep it `TIERED`, there is no need to rename it externally

File: src/server/server_family.cc [hunk 2]
```diff
@@ -2056,24 +2047,20 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
...
     append("tiered_v2_total_stashes", m.tiered_stats_v2.total_stashes);
     append("tiered_v2_total_fetches", m.tiered_stats_v2.total_fetches);
+    append("tiered_v2_total_cancels", m.tiered_stats_v2.total_cancels);
```
Comment (romange) [inline src/server/server_family.cc:2053]: similarly the variable names and the external metrics

File: src/server/server_family.cc [hunk 3]
```diff
@@ -2057,23 +2048,19 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
...
-    append("tiered_aborted_writes", m.tiered_stats.aborted_write_cnt);
-    append("tiered_flush_skipped", m.tiered_stats.flush_skip_cnt);
-    append("tiered_throttled_writes", m.tiered_stats.throttled_write_cnt);
-  }
-
-  if (should_enter("TIERED_V2", true)) {
-    append("tiered_v2_total_stashes", m.tiered_stats_v2.total_stashes);
-    append("tiered_v2_total_fetches", m.tiered_stats_v2.total_fetches);
-    append("tiered_v2_allocated_bytes", m.tiered_stats_v2.allocated_bytes);
+    append("tiered_total_stashes", m.tiered_stats_v2.total_stashes);
```
Comment (romange) [inline src/server/server_family.cc:2051]: if you removed `m.tiered_stats`, you can also rename tiered_stats_v2 to tiered_stats ;)

**Review by romange — APPROVED** [review]

**Pull request merged**